faqs.md

```diff
@@ -0,0 +1,155 @@
+
+This Galaxy Gmentor Network Programme helps participants (network managers, Gmentors, Gmentees) to:
+- share their expertise and develop skills and knowledge vital to create, lead, and contribute to Galaxy projects.
+- connect with members across different projects, communities, backgrounds, and identities.
+- empower each other to become effective Galaxy community members.
+
+### How to join the program?
+**As part of the process:**
+
+Gmentees join this program with goals that they want to achieve during this program. With support from their mentors, they complete these successfully.
+
+Gmentors join the program by indicating their availability and willingness to mentor someone.
+
+Network Managers accept to be part of the program by replying to the invite message.
+
+### Can you tell me more about the Gmentors-Gmentees-Pairing (MMP) Process?
+Once a Gmentee submitted an application, it will be reviewed by the network manager and then shared with the pool of possible Gmentors. The Gmentors select their own Gmentees together with the network manager. The areas of Gmentorship are not limited to: Contributing to the community, Running data analysis, Instructing students, and Carrying out projects. This opportunity will help them over a period of two months to develop new skills and expertise and to be proficient and grow with the ecosystem.
+
+There are 2 general rules: 
+- Discussions and everything you will share with your Gmentor will remain between you (except in case of Code of Conduct violation)
+- Gmentor & Gmentee schedule a minimum of a monthly virtual meeting.
+
+Each Gmentor/Gmentee is provided with a toolkit (slide deck, short video, and note-taking document) at the beginning of the programme. The programme is at hand to help throughout the process.
+
+### What happens at the end of the two months of Gmentoring? Can I continue my Gmentorship relationship?
+Of course! Beyond the two months, Gmentor/Gmentee relationships continue based on the mutual interest of both parties. Officially, the program winds down after two months, but some of the best Gmentoring relationships continue well past the end of any formal, organizationally sanctioned timeframe.
+
+### What happens if I need to leave the programme before it is concluded?
+It is not recommended to leave the programme before it is concluded, as it is a unique opportunity and experience that will add value to you from the beginning to the end. Its flexible nature allows you to accommodate other responsibilities while participating in the programme. If, however, you have to leave the programme, please notify us immediately so that someone else can be matched with your mentoring partner.
```

Comment:
```suggestion

It is not recommended to leave the programme before it is concluded, as it is a unique opportunity and experience that will add value to you from the beginning to the end. Its flexible nature allows you to accommodate other responsibilities while participating in the programme. If, however, you have to leave the programme, please notify us immediately so that someone else can be matched with your mentoring partner.
```